_pages/infrastructure.md

```diff
@@ -12,6 +18,37 @@ title: Infrastructure
 
 As a result, it will reduce the portion of your team’s capacity that you need to dedicate to operational concerns. For this reason, when making infrastructure decisions, **opt to use cloud.gov for your deployment whenever possible, and only resort to directly using AWS for infrastructure pieces that are impossible to achieve through cloud.gov**.
 
-There are some specific cases where cloud.gov may not be a good choice for your project. This is particularly the case when the product is classified “FISMA High”. This would usually only happen due to your product handling extremely sensitive information  or being critical to normal government function. FISMA High products require wholly dedicated infrastructure, so neither cloud.gov nor AWS are options for deploying them.
+#### Amazon Web Services (AWS)
+
+If you do want to use AWS directly, see the [AWS](aws/) page.
+
+#### FISMA High systems
+
+There are some specific cases where the product is [classified](../ato/levels/) “FISMA High”. This would usually only happen due to your product handling extremely sensitive information or being critical to normal government function. FISMA High products require wholly dedicated infrastructure, so neither cloud.gov nor AWS are options for deploying them.
 
 Note however that when partner agencies assert that **of course** their product will be FISMA High, 18F often finds upon examination that a product should be really be judged FISMA Moderate or FISMA Low... So don’t discard cloud.gov or AWS as options before probing that point carefully!
+
+### Static sites
+
+There are several options for deploying static sites at 18F:
+
+#### cloud.gov
+
+See cloud.gov page on [deploying static sites](https://docs.cloud.gov/apps/static/).
+
+* **Internal:** free
+* **External:** see the [pricing page](https://docs.cloud.gov/intro/pricing/pricing-model/)
+
+#### Federalist
+
+See [the Federalist homepage](https://federalist.18f.gov) for more information.
+
+* **Internal:** free
```

Comment:
Are you thinking for things like Great Pitch projects? Maybe changing the definition of "internal" above to be "paid for out of 18F overhead"? I defer to you on this!